src/Microsoft.Diagnostics.DebugServices/IRuntimeService.cs

```diff
@@ -21,16 +21,5 @@ public interface IRuntimeService
         /// Returns the list of runtimes in the target
         /// </summary>
         IEnumerable<IRuntime> EnumerateRuntimes();
```

Comment:
That is the main point of this PR: it to remove any kind of "context" in the core services but I couldn't decide to move this property to the context services or not. 

It would be harder to use to move it to IRuntime for the setclrpath command.  I guess the command could just assume the current runtime.  This would allow it to be set differently between a .NET Core and .NET Framework runtimes in the same process.  I need to think about this one also.

Reply:
I think it does make sense to move this property to IRuntime.